v1.1/deploy-cockroachdb-on-aws-insecure.md

```diff
@@ -264,6 +266,13 @@ To test this, install CockroachDB locally and use the [built-in SQL client](use-
 	(1 row)
 	~~~
 
+    The `node_id` [session variable](show-vars.html) used above is an alias for the following query, which you can use as well:
```

Comment:
This is not true any more. 

Also, as a general principle, I would not recommend / advertise `crdb_internal` in user-facing docs. It's called "internal" for a reason :)
If you find over time that some features available via `crdb_internal` tables happen to also be useful to users, let the eng team know so we can devise a properly advertisable feature outside of the "internal" namespace.